lib/utils.pm

```diff
@@ -1267,13 +1267,11 @@ sub check_prerelease {
 
 sub check_version {
     # This function checks if the correct version is display during installation
-    # in Anaconda, i.e. nonlive media showing Rawhide when Rawhide and version numbers
-    # when not Rawhide, while live media always showing version numbers.
+    # in Anaconda, it only looks at the major version, as these needles should
+    # not change between the major versions.
 
     my $version = lc(get_var('VERSION'));
-    if ($version eq 'rawhide' && get_var('LIVE')) {
-        $version = get_var('RAWREL');
-    }
+    $version =~ s/\..+$//;
```

Comment:
I think a more complete test here is to drop to the root console and detect the release outside the GUI. This will get us past this hurdle but we want to ensure the installer we run is actually for the VERSION we are testing as our openQA needs to work for all supported versions at the same time.